### PR TITLE
Fluid datepicker v2

### PR DESCRIPTION
**Pull request by aagonzales**
Closes https://github.com/carbon-design-system/carbon-website/issues/3190 
Closes https://github.com/carbon-design-system/carbon/issues/12476

Fluid DatePicker and Fluid TimePicker documentation update.

#### Changelog

- Updated Usage tab to include Fluid content
   - Added "Styling" section under Formatting
   - Added Fixed/Fluid tabs to certain images
- Updated Style tab to include Fluid content

### CODE REVIEW

**Comment by alisonjoseph** [comment]
@aagonzales I can help with the merge conflicts, but I think I'll need permissions on your fork to do anything. 

https://github.com/aagonzales/carbon-website/  Settings > Collaborators > Add Collaborator

**Comment by aagonzales** [comment]
@alisonjoseph ok I invited you

**Comment by alisonjoseph** [comment]
@aagonzales should be resolved 🙏

**Comment by aagonzales** [comment]
Thank you @alisonjoseph!!

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by alisonjoseph — APPROVED** [review]

File: src/pages/components/date-picker/usage.mdx [hunk 1]
```diff
@@ -240,43 +254,85 @@ You can customize the time picker's format depending on location or need.
...
+![Fixed input size example](images/date-picker-sizes.png)
 
 </Column>
 </Row>
 
-### Sizes
+#### Fluid input heights
+
+There is only one fluid input height and it is visually larger than the fixed
+heights. The input is a set height of 64px expect when a warning or error
```
Comment (laurenmrice) [inline src/pages/components/date-picker/usage.mdx:300]:
```suggestion
heights. The input is a set height of 64px except when a warning or error
```

File: src/pages/components/date-picker/usage.mdx [hunk 2]
```diff
@@ -240,43 +254,85 @@ You can customize the time picker's format depending on location or need.
...
+</Column>
+</Row>
+
+### Alignment
+
+By default, the pickers have set widths. If you are placing the picker inline
+with other inputs, such as in form, then the widths can be adjusted to match the
+other inputs. The picker can either increase or decrease in width as needed. If
+you adjust the size, be aware that pickers have minimum widths and the date
+content should never horizontal scroll or overflow.
```
Comment (laurenmrice) [inline src/pages/components/date-picker/usage.mdx:317]:
```suggestion
content should never horizontally scroll or overflow.
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks great! ⭐️ Just a couple of small things:

### Usage tab

**Styling section**
- In the image the label specifies (`mm/dd/yyy`). It is missing a fourth `y`. 
Screen Shot 2022-11-14 at 2 28 25 PM

- The cropped image of date picker menu needs border rules so it doesn’t blend in with the website background.
Screen Shot 2022-11-14 at 2 28 47 PM

**Alignment section**
- Image needs bottom border rules so it doesn’t blend in with the website background.
Screen Shot 2022-11-14 at 2 29 18 PM

**States section**
- Fluid image: The disabled state input has a different date than the other dates. There fixed disabled state image just has placeholder text. I would either do placeholder text or keep the same 05/19/2022 date for consistency. (This also appears in an image on the style tab)

**Comment by aagonzales** [comment]
Ok @laurenmrice I made those updates. I also went ahead and actually added TimePicker to this PR since it was still open if you want to check that content as well.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
**Usage tab**
- [Time pickers section](https://carbondesignsystem-git-fork-aagonza-8a78b6-carbon-design-system.vercel.app/components/date-picker/usage#time-pickers): This was a current bug in the docs, but this sentence needs to have a word added to it

> The time field format should include the hour and minutes, for example 11:30. It may be localized `to` accommodate the 12-hour or 24-hour format.


**Style tab**
- This spec image for Time picker needs to have `$text-primary` warning text.
Screen Shot 2022-11-17 at 2 28 21 PM

- This spec image for Time picker need to have `$text-primary` for the "Timezone" dropdown text.
Screen Shot 2022-11-17 at 2 31 53 PM

**Review by laurenmrice — APPROVED** [review]
🎉

**Pull request merged**